### ISSUE TITLE
Make ctrl+scroll zoom in/out FFT

### ISSUE THREAD

**Issue by d33tah** [csete/gqrx#387]
Sometimes I'd like to be able to zoom in in gqrx and switching to "FFT settings" tab to change that isn't too conivenient. Could you bind zoom in/out behavior to ctrl+scroll?

**Comment by csete** [comment]
You can use the mouse wheel on the frequency and dB axis to zoom. You can also drag the axes to pan. The widget in the FFT tab were only added recently in case mouse zooming works bad with some touch pads.

**Comment by d33tah** [comment]
Ah! Nice, that solves the problem for me. Though from UX perspective, adding the ctrl+scroll might still be pretty intuitive.

**Comment by csete** [comment]
I assume with "UX perspective" you mean something like a standard image viewing application or map  where ctrl+scroll will zoom in and out. Ok, fine... I understand that. However, in SDR zooming both X and Y axes at the same time makes little sense, at least in my opinion.

So, what should ctrl+scroll zoom, X or Y?

**Comment by csete** [comment]
Perhaps we can use ctrl+scroll for zoom in frequency and shift+scroll to zoom in dB?

**Comment by d33tah** [comment]
@csete: yup, that makes sense. shift+scroll for "db range", ctrl+scroll for "zoom" from FFT settings tab.

**Comment by Analias** [comment]
FYI - Ctrl-Scroll (I assume "Scroll" refers to the scroll wheel on the mouse) on the Mac is used to zoom the entire Mac desktop. I don't see a way of overriding the default behavior for the Mac.

**Comment by Analias** [comment]
I did a little bit more research - it is possible to disable the UI Ctrl-Scroll on the Mac in the Accessibility system preferences under the Zoom settings. You can change the keyboard modifier from Ctrl to Option or Command key, but that's not very useful in this use case.  Disabling the UI scrolling re-enables the gqrx Ctrl-Scroll.

**Comment by csete** [comment]
Thanks for the info @Analias . It looks like these desktop zoom shortcuts are disabled by default.
